RFC/0001-lightweight-rfc-process.md

```diff
@@ -0,0 +1,117 @@
+# RFC 0001: Lightweight RFC Process for IPFS Specifications
+
+- Start Date: 2022-06-10
+- Related Issues:
+  - [ipfs/specs/issues/286](https://github.com/ipfs/specs/issues/286)
+
+# Summary
+
+This Request for Change (RFC) introduces a lightweight RFC process
+for the IPFS specifications [repository][1].
+
+[1]: https://github.com/ipfs/specs/
+
+# Motivation
+
+Today, protocol design discussions often take place in a repository of an IPFS
+implementation. These conversations are unintentionally obscured from the useful input of IPFS Stewards, other
+implementations, service operators and the wider IPFS Community.
+
+The IPFS Project needs a mechanism for proposing and evaluating specification
+improvements that are not tied to a specific programming language
+or implementation of IPFS.
+
+# Detailed design
+
+Adopt an informal RFC process for the [ipfs/specs][1] repository, providing a
+minimal structure for opening, reviewing, and merging specification changes.
+
+## Opening
+
+Changes to IPFS specifications can be proposed by opening a PR against the
+repository, and including a short "request for change" document based on the
+template in `RFC/0000-template.md`.
+
+## Reviewing
+
+IPFS Stewards will review new RFC PRs during weekly triage.
+
+IPFS Community is encouraged to participate in the review process.
+
+Final decision belongs to IPFS Stewards.
+
+## Merging
+
+RFC can be merged only after two Stewards approve it.
```

Comment:
- What does it mean to be merged?
- What does it mean if an RFC is closed?

Reply:
I clarified we mean a git pull-request with changes that include an improvement proposal document (IPIP).